### ISSUE TITLE
Compatible with OpenBSD?

### ISSUE THREAD

**Issue by anoduck** [conda/conda#10074]
### Compatibility with OpenBSD (or any other bsd for that matter)?
It appears as if conda is no longer available with OpenBSD or any other bsd. There is no build instructions or information on how to use conda with these distros where there once was.

**Comment by anoduck** [comment]
Miniconda is compatible, and has been installed on the system previously. At the moment, poetry, pipx, and pipenv are facilitating the functionality that conda was intended for. 

So, this issue is overdue for being closed out.

**Comment by anoduck** [comment]
@jamespolly
__...Enter the Duck...__

It has been some time now, but miniconda was doable at some point in time, or at least I thought it was. I do remember running the "conda" command, so it must have been. 

It appears the method for which one installs conda has changed significantly from when I was running it. In it's current form, a 98MB bash script with what looks like inline binary, tampering with the bash script will not be a successful strategy, and will only result in a binary that is not compatible with BSD. On my system, opening a 98MB shell script while running bash-lsp will crash my Neovim instance, oddly EMACS can take it. 

So, what needs to happen is for me to reopen this issue, and for us to work with developers to provide a reasonable resolution for other non-linux systems. It used to not be like this, in the past, BSDs were just as valued as linux in the open source community. Windon't and MacOS were lesser valued, but now this is not the case.

**Comment by anoduck** [comment]
@jamespolly Whatever happened to repositories actually possessing buildable source code? Every method of installation I have found is dependent on that 98MB 'Linux only' build script.

**Comment by anoduck** [comment]
@chenghlee How do you build conda if you are not running one of the big three OSs? There has to be a way to allow users of the BSDs to use it. 

And just for the sake of curiosity, is that inline binary in the installation script?

You used to be able to get miniconda running on OpenBSD, but it appears the method of installation has changed since that point.

**Comment by jamespolly** [comment]
@anoduck Thanks for your input and FWIW I agree. I've looked around for docs on how to build miniconda "from source" but the large shell script is all I've found.

I'll add that a FreeBSD [port](https://www.freshports.org/sysutils/linux-miniconda-installer/) exists which [simply](https://github.com/outpaddling/miniconda-installer/blob/main/miniconda-installer) enables a linux compatibility layer within FreeBSD and runs the miniconda installer script. We need something lower level.
